docs/en/sql-reference/statements/delete.md

```diff
@@ -7,57 +7,81 @@ keywords: [delete]
 title: DELETE Statement
 ---
 
+The `DELETE` statement removes rows from the table `[db.]table` that match the expression `expr`.
+
 ``` sql
-DELETE FROM [db.]table [ON CLUSTER cluster] WHERE expr
+DELETE FROM [db.]table [ON CLUSTER cluster] WHERE expr;
 ```
 
-`DELETE FROM` removes rows from the table `[db.]table` that match the expression `expr`. The deleted rows are marked as deleted immediately and will be automatically filtered out of all subsequent queries. Cleanup of data happens asynchronously in the background. This feature is only available for the MergeTree table engine family.
+Deleted rows are internally marked as deleted immediately and will be automatically filtered out of all subsequent queries. Cleanup of data happens asynchronously in the background. The `DELETE` statement is only available for the *MergeTree table engine family.
+
+**Do not use the DELETE statement frequently as it can negatively affect ClickHouse performance.**
```

Comment:
I am also not sure about this statement because yes, it won't really affect the performance as much as it will just take an undefined amount of time to delete the data.

Reply:
What's a better way to phrase this?

**Deleting large volumes of data frequently can negatively affect ClickHouse performance.**

Or should we just omit this?

Reply:
Maybe something like this?
```
Deleting large volumes of data with the DELETE statement can negatively affect SELECT query performance.
```